vpr/src/timing/timing_util.cpp

```diff
@@ -579,6 +579,23 @@ float calculate_clb_net_pin_criticality(const SetupTimingInfo& timing_info, cons
     return clb_pin_crit;
 }
 
+//Return the setup slack of a net's pin in the CLB netlist
```

Comment:
Explain that it assumes the timing analysis is up to date (has already been performed with an update call with the right delays).